src/containers/ImportResources/ImportResources.js

```diff
@@ -159,7 +159,7 @@ export class ImportResources extends Component {
           />
           <TextInput
             data-testid="directory-field"
-            helperText="The path from which resources will be applied at the specified repository"
+            helperText="The location of the Tekton resources in the repository (Leave blank if resources at root)."
```

Comment:
Big nit but... `(Leave blank if resources at root).` is still a bit confusing, and it should be a lower case for `Leave`

How about
"The location of the Tekton resources to import from the repository. Leave blank if the resources are at the top-level directory.`?